src/main/resources/CodyBundle.properties

```diff
@@ -52,6 +52,10 @@ chat.rate-limit-error.explain=\
     <a href=\"https://sourcegraph.com/docs/cody/core-concepts/cody-gateway#rate-limits-and-quotas\">Learn more.</a>\
   </html>
 
+subscription-tab.chat-rate-limit-error=You've used all your chat messages and commands for the month. Upgrade to Pro for unlimited usage.
```

Comment:
The overall experience is much better now without the flickering effect 🎉 

I have two points:
- [ ] No notification for a **pro account**:
In all other places we provide some handling even for the pro rate-limited accounts (however, this scenario seems unreal). 

- [ ] When I run a recipe the message does not appear.

Reply:
I think we won't show the notification for pro user accounts. Doesn't make sense since pro has unlimited access...

Reply:
but we have a testing account that is pro & unlimited so it's easy to test + we handle that case everywhere else